app/src/main/res/values/strings.xml

```diff
@@ -1,9 +1,9 @@
 <resources>
 
     <!-- General -->
-    <string name="example_title">ExampleFragment</string>
-    <string name="example_textview">Some example TextView</string>
-    <string name="example_randomize">Randomize</string>
-    <string name="example_wolox">WOLOX</string>
-    <string name="example_message">The presenter generated the number %1$d and this view is displaying it</string>
+    <string name="login_email_address">Email address</string>
+    <string name="login_password">Password</string>
+    <string name="login_login">LOG IN</string>
```

Comment:
You can write the text like Log in and then set the uppercase to the button . Then you can use the same string resource in other components :)